### DISCUSSION TITLE
Free Shipping on products does not work in combination with order-level shipping rates

### DISCUSSION THREAD

**Discussion by RandomJo** [craftcms/commerce#2333] [Ideas]
Hey, Luke. I tested this out in the develop branch, and the new discount option works well—just as expected. However, the free shipping checkbox on the product doesn't work like I was anticipating. I have a tiered base rate for shipping based on the Maximum Order Total Value and Minimum Order Total Value fields. But, when I mark free shipping on a product, it is not removed from that calculation. The base rate for shipping still applies even when I have one item in the cart that is marked for free shipping.

I also tried setting a minimum value of $0.01 on the lower shipping tier to see if that made a difference and it didn't.

Should I expect the free shipping checkbox to do anything in this scenario, or did I misunderstand the change you made?

_Originally posted by @RandomJo in https://github.com/craftcms/commerce/issues/561#issuecomment-465789263_

**Comment by lukeholder** [comment]
At the moment if a product has been marked as free shipping, that doesn't affect the min and max "Total Order Value". 

"Value" is the key word here, as it is about the cost of the items before any discounts/tax or shipping cost.

We could look to add an option that say "Don't include free shipping items in value total" or something to that affect.

We also should be introducing a "inclusive" checkbox next to min and max conditions to allow a `>=` in addition to a `>`. 

Leaving this ticket open and will be on the list.

**Comment by RandomJo** [comment]
This is great. Thank you, Luke!

**Comment by echantigny** [comment]
@lukeholder 

I'm also looking at a way of offerring Free Shipping on certain items.  Right now, there doesn't seem to be a way to do this at all.  The Free Shipping check is not working and creating a Shipping category for those products makes it a stale mate if a product of each is in the cart.  Not even sure how the Shipping Category Conditions work either.
